components/edit-collective/sections/virtual-cards/VirtualCards.js

```diff
@@ -170,7 +185,16 @@ const VirtualCards = props => {
       </Box>
       <Grid mt={4} gridTemplateColumns={['100%', '366px 366px']} gridGap="32px 24px">
         {data.account.virtualCards.nodes.map(vc => (
-          <VirtualCard key={vc.id} {...vc} />
+          <VirtualCard
+            canEditVirtualCard={
+              props.collective.type === CollectiveType.USER &&
+              vc.provider === 'STRIPE' &&
+              (vc.data.state === 'OPEN' || vc.data.status === 'active')
+            }
+            showConfirmationModal={props.collective.type === CollectiveType.USER}
```

Comment:
I don't understand what this is.

Reply:
Do we have 2 variables with the same name? One being a prop and one being a state? Highly confusing.

Reply:
I've renamed the conflicting prop and state, with the prop name as, `confirmOnPauseCard`